EED_Multi_Event_Registration.module.php

```diff
@@ -2212,6 +2217,20 @@ public static function save_cart() {
 	}
 
 
+	/**
+	 * setInvisiblerecaptchaLocalizedVars
+	 * This filters \EventEspresso\caffeinated\modules\recaptcha_invisible\InvisibleRecaptcha::getLocalizedVars() 
+	 * to set disable_submit to true.
+	 *
+	 * @access public
+	 * @param array $localizedVars
+	 * @return array
+	 */
+	public static function setInvisiblerecaptchaLocalizedVars($localizedVars) {
```

Comment:
we don't need to declare function scopes in phpdocs anymore, modern IDEs can determine the scope and it's in the function declaration just under the docs.
as well, it's best to try and type cast for parameters as much as possible because it massively cuts down on errors that can occur due to data type mismatches.

```suggestion
	 * @param array $localizedVars
	 * @return array
	 */
	public static function setInvisiblerecaptchaLocalizedVars(array $localizedVars) {
```